config/jackal.lua

```diff
@@ -19,12 +19,12 @@ options = {
   map_builder = MAP_BUILDER,
   trajectory_builder = TRAJECTORY_BUILDER,
   map_frame = "map",
-  tracking_frame = "imu_link",
+  tracking_frame = "base_link",
   published_frame = "odom",
   odom_frame = "odom",
   provide_odom_frame = false,
   publish_frame_projected_to_2d = false,
-  use_odometry = true,
+  use_odometry = false,
```

Comment:
How come you disabled the odometry?

Reply:
No reason really.. was testing it without and forgot to change it back. Both configs produced good results

Reply:
Ah okay, thanks for adding this back in.